mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -360,7 +361,29 @@
      * @return              List<MailboxAnnotation>
      * @throws MailboxException in case of selected mailbox does not exist
      */
-    List<MailboxAnnotation> getAnnotationsByKeys(MailboxPath mailboxPath, MailboxSession session, Set<String> keys) throws MailboxException;
+    List<MailboxAnnotation> getAnnotationsByKeys(MailboxPath mailboxPath, MailboxSession session, Set<MailboxAnnotationKey> keys) throws MailboxException;
+
+    /**
+     * Return all mailbox's annotation by the list of the keys and its children entries without order and not duplicated by key
```

Comment:
It means: "the returned result does not contain any two mailbox's annotations with the same key". So it should be:
".. and do not contain any two mailbox's annotations with the same key". 
Is that right?